erpnext/stock/doctype/item/item.json

```
@@ -1055,17 +1054,17 @@
   },
   {
    "default": "0",
-   "fieldname": "inspection_required_before_manufacturing",
+   "fieldname": "inspection_required_during_manufacturing",
    "fieldtype": "Check",
-   "label": "Inspection Required before Manufacturing"
+   "label": "Inspection Required During Manufacturing"
```

This is a nitpick, but keep it consistent with the other inspection check fields:

```suggestion
   "label": "Inspection Required during Manufacturing"
```